src/Libraries/CoreNodeModels/Remember.cs

```diff
@@ -0,0 +1,126 @@
+using Dynamo.Graph;
+using Dynamo.Graph.Nodes;
+using Newtonsoft.Json;
+using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using VMDataBridge;
+
+namespace CoreNodeModels
+{
+    [NodeName("Remember")]
+    [NodeDescription(nameof(Properties.Resources.RememberDescription), typeof(Properties.Resources))]
+    [NodeCategory("Core.Data")]
+    [InPortNames(">")]
+    [InPortTypes("object")]
```

Comment:
var?

Reply:
This node does not replicate.  We can change the label to whatever we would like.  In the Node model case the port names are just for display.  What do you think is the oppriate label @mjkkirschner

Reply:
I think the appropriate label for a none replicating node is `var[]..[]`